main.tf

```diff
@@ -43,6 +43,24 @@ resource "aws_guardduty_detector" "primary" {
   }
 }
 
+##################################################
+# GuardDuty Features Configuration
+##################################################
+resource "aws_guardduty_detector_feature" "this" {
+  for_each    = var.configuration_features
+  detector_id = aws_guardduty_detector.primary.id
+  name        = each.name
+  status      = each.status ? "ENABLED" : "DISABLED"
```

Comment:
wont this set `status` to `ENABLED` if I set `var.configuration_features.status = "disabled"` ?